patches/api/0002-Build-system-changes.patch

```diff
@@ -37,16 +37,19 @@ index 4b7f005519d8856fc35a94401636ae8d9bf41f10..bb6e3afe8e8bc1a3b598b085286ffcda
      testImplementation("org.apache.commons:commons-lang3:3.12.0")
      testImplementation("junit:junit:4.13.2")
      testImplementation("org.hamcrest:hamcrest-library:1.3")
-@@ -65,7 +77,7 @@ tasks.withType<Javadoc> {
+@@ -65,8 +77,10 @@ tasks.withType<Javadoc> {
      options.links(
          "https://guava.dev/releases/31.1-jre/api/docs/",
          "https://javadoc.io/doc/org.yaml/snakeyaml/1.33/",
 -        "https://javadoc.io/doc/org.jetbrains/annotations-java5/23.0.0/",
 +        "https://javadoc.io/doc/org.jetbrains/annotations/23.0.0/", // Paper - we don't want Java 5 annotations
          "https://javadoc.io/doc/net.md-5/bungeecord-chat/1.16-R0.4/",
++        "https://javadoc.io/doc/org.apache.logging.log4j/log4j-api/2.17.1/",
++        "https://javadoc.io/doc/org.slf4j/slf4j-api/1.8.0-beta4",
```

Comment:
Not a required change, but should we make `val` fields for the versions for these 2 dependencies? just to ensure they are consistent with the dependencies added in the dependency block.